lib/bulk_reindexer.rb

```diff
@@ -0,0 +1,15 @@
+module Argo
+  class BulkReindexer
+    def self.queue_pid_reindexing_jobs(pid_list, should_commit = false, should_profile = false, priority = 1, batch_size = 10)
```

Comment:
if I understand this correctly, you have `@param [Array<String>] pid_list`, and `get_pid_lists_for_full_reindex` returns an `Array<Array<String>>`. it's a little dense here in this important code and could use some comments.